skimage/segmentation/tests/test_expand_labels.py

```diff
@@ -0,0 +1,189 @@
+from itertools import product
+from scipy import ndimage as ndi
+from skimage import data
+
+import numpy as np
+from numpy import array
+
+from skimage import measure
+from skimage.segmentation._expand_labels import expand_labels
+
+from skimage._shared import testing
+from skimage._shared.testing import assert_array_equal
+
+SAMPLE1D = np.array([0, 0, 4, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0])
+SAMPLE1D_EXPANDED_3 = array([4, 4, 4, 4, 4, 4, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0])
```

Comment:
Please be consistent with previous line ;)
```suggestion
SAMPLE1D_EXPANDED_3 = np.array([4, 4, 4, 4, 4, 4, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0])
```

Reply:
thanks for spotting this copy&paste error